archivers/libzzip/Portfile

```diff
@@ -64,14 +57,43 @@ post-destroot {
 }
 
 variant sdl description {Enable SDL support} {
-    depends_lib-append      port:libsdl
+    depends_lib-append      port:libsdl2
 
-    configure.args-append   --enable-sdl
+    configure.args-append   -DZZIPSDL=ON
```

Comment:
I believe `ZZIPSDL` defaults to on in this project's CMakeLists.txt. Therefore you have to set `-DZZIPSDL=OFF` in the global part of the portfile, and here, replace it with `-DZZIPSDL=ON`.

Reply:
You're right. Will fix now.